shell/platform/darwin/ios/platform_view_ios.h

```diff
@@ -66,6 +66,11 @@ class PlatformViewIOS final : public PlatformView {
    */
   void SetOwnerViewController(__weak FlutterViewController* owner_controller);
 
+  /**
+   * Send accessibility message to accessibility channel.
+   */
+  void sendAccessibilityMessage(__weak id message);
```

Comment:
```suggestion
  void SendAccessibilityMessage(__weak id message);
```
Need to fix up the call sites.